contracts/src/spaces/facets/membership/metadata/MembershipMetadata.sol

```diff
@@ -2,6 +2,7 @@
 pragma solidity ^0.8.23;
 
 // interfaces
+import {IERC4906} from "@openzeppelin/contracts/interfaces/IERC4906.sol";
```

Comment:
Where is this being used?

Reply:
It's not being used because it conflicts with `IERC721ABase`. Will need a refactor later.